chapters/develop-organize/reading/reading/README.md

```diff
@@ -18,8 +18,12 @@ To enforce the guidelines we use [automated linter checks](https://github.com/su
 Besides the mentioned guidelines, the linter implements checks for other minor style conventions (such as how to define lists, where should new lines be placed and others).
 However, for the majority of cases, the above should be sufficient.
 
+The reading content type is stored as a folder named `reading` in the course repository.
+Each topic should contain a `reading` folder that includes one or multiple Markdown files, depending on how the topic may be broken down in independent reading parts.
+
 When developing such materials we recommend that:
 
 - the reading material be concise, following the "less is more" principle.
 - the first sentence of each section describe what is the intended purpose of the section.
 - if the reading material can be expressed in a more visual manner, the latter is preferred.
+- the first person plural is used for explanations: "we derive x", "we download the application" etc.
```

Comment:
I would move this up, before the markdown syntax list.

Reply:
I think it's better this way since all that comes before this list is can be categorized as "objective requirements" (i.e. we can automatically check for them) whereas this list is quite subjective and open to debate.

Reply:
Fair point, we can leave it like it is now.